src/procedure/nodes/sizeFactor.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#include "procedure/nodes/sizeFactor.h"
+#include "classes/configuration.h"
+#include "keywords/nodeValue.h"
+#include "procedure/nodes/node.h"
+SizeFactorProcedureNode::SizeFactorProcedureNode()
```

Comment:
Arrgggh - it's the Blank Line Police again! :smile:
```suggestion

SizeFactorProcedureNode::SizeFactorProcedureNode()
```